PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -536,22 +550,24 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
 (run3_nanoAOD_devel).toModify(slimmedElectronsWithUserData.userFloats,
                                                     mvaFall17V1Iso = None,
                                                     mvaFall17V1noIso = None,
-                                                    mvaFall17V2Iso = None,
-                                                    mvaFall17V2noIso = None,
+                                                    mvaIso = None,
+                                                    mvaNoIso = None,
+                                                    mvaHZZIso = None,
                                                 )
 (run3_nanoAOD_devel).toModify(electronTable.variables,
-                              mvaFall17V2Iso = None,
-                              mvaFall17V2Iso_WP80 = None,
-                              mvaFall17V2Iso_WP90 = None,
-                              mvaFall17V2Iso_WPL = None,
-                              mvaFall17V2noIso = None,
-                              mvaFall17V2noIso_WP80 = None,
-                              mvaFall17V2noIso_WP90 = None,
-                              mvaFall17V2noIso_WPL = None,
+                              mvaIso = None,
+                              mvaIso_WP80 = None,
+                              mvaIso_WP90 = None,
+                              mvaIso_WPL = None,
+                              mvaNoIso = None,
+                              mvaNoIso_WP80 = None,
+                              mvaNoIso_WP90 = None,
+                              mvaNoIso_WPL = None,
                               vidNestedWPBitmapHEEP = None,
                               vidNestedWPBitmap = None,
                               cutBased = None,
                               cutBased_HEEP = None,
+                              mvaHZZIso = None,
```

Comment:
mvaHZZIso = None,
For the first nano-version of the Run3, do you want to have one variable defined with the Run2 training in i.e. that of the 2018 ? later you modify

Reply:
That was the initial idea, but as @swagata87 pointed out in her reply:

> Run3-specific failures ring a bell.. Ana, you might need to also add your IDs in the following 2 places inside `PhysicsTools/NanoAOD/python/electrons_cff.py`;
> 
> here:
> 
> https://github.com/cms-sw/cmssw/blob/0cd6c4f6a6ac8034ffbaff4eb6e2725d820c216e/PhysicsTools/NanoAOD/python/electrons_cff.py#L536-L540
> 
> and just in the next block, i.e. here:
> 
> https://github.com/cms-sw/cmssw/blob/0cd6c4f6a6ac8034ffbaff4eb6e2725d820c216e/PhysicsTools/NanoAOD/python/electrons_cff.py#L542-L555
> 
> does this make sense?

these Run3-specific failures were solved before by simply removing those IDs in Run3, I am not sure if it is possible to have them defined with Run2 training without causing failures.  When I try to set it to Run2 value and test it fails with 

```
----- Begin Fatal Exception 27-May-2022 10:55:55 CEST-----------------------
An exception of category 'InvalidReference' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 2 stream: 0
   [1] Running path 'dqmoffline_3_step'
   [2] Prefetching for module NanoAODDQM/'nanoDQMMC'
   [3] Prefetching for module BTagSFProducer/'btagWeightTable'
   [4] Prefetching for module PATObjectCrossLinker/'linkedObjects'
   [5] Prefetching for module PATElectronRefSelector/'finalElectrons'
   [6] Calling method for module PATElectronUserDataEmbedder/'slimmedElectronsWithUserData'
Exception Message:
ValueMap: no associated value for given product and index
----- End Fatal Exception -------------------------------------------------
```